src/components/obs/ObsHistory.jsx

```diff
@@ -73,8 +73,10 @@ class ObsHistory extends React.PureComponent {
 
   }
 
-  isEditableEncounter(encounter) {
-    return encounter && encounter.encounterType && this.props.editableEncounterTypes.map(e => e.uuid).includes(encounter.encounterType.uuid);
+  isEditableEncounter(encounter, obs) {
```

Comment:
This function determines if an encounter is editable or not (which determines whether or not we display the pencil and hyperlink)

Previously, it simply checked to see if the encounter type was found in the "editableEncounterTypes" prop but I've added a editableFunc prop which allows a developer to define a custom function to determine if a specific encounter is editable or not... this function gets passed both the encounter and the obs associated with the encounter.

Reply:
Won't there be multiple obs associated with the encounter? How do I know which obs to pass in? Maybe a function comment to explain would be good?

Reply:
That's really a list of obs... it's annoying that "obs" ends in a "s", I'll change to obsLIst for clarity.